### PR TITLE
[MBO-1372] MBO-4 : Integrate and hide Accounts component

### PR DESCRIPTION
**Pull request by sowbiba**

### CODE REVIEW

File: config/services/accounts.yml [hunk 1]
```diff
@@ -2,6 +2,18 @@ services:
   _defaults:
     public: true
 
+  mbo.ps_accounts.installer:
+    class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
+    public: true
+    arguments:
+      - "5.2.5"
```
Comment (intraordinaire) [inline config/services/accounts.yml:9]: Maybe we can put the 6.X version here as we are on a PS > 8.

File: src/Service/View/ContextBuilder.php [hunk 2]
```diff
@@ -177,6 +177,7 @@ private function getCommonContextContent(): array
             'accounts_user_id' => $this->accountsDataProvider->getAccountsUserId(),
             'accounts_shop_id' => $this->accountsDataProvider->getAccountsShopId(),
             'accounts_token' => $this->accountsDataProvider->getAccountsToken(),
+            'accounts_component_loaded' => false,
```
Comment (intraordinaire) [inline src/Service/View/ContextBuilder.php:180]: Always to false ?

**Review by intraordinaire — APPROVED** [review]

File: src/Service/View/ContextBuilder.php (hunk 2 above)
Comment (sowbiba) [reply src/Service/View/ContextBuilder.php:180]: The value is changed when the component is actually loaded in the template

https://github.com/PrestaShopCorp/ps_mbo/pull/670/files#diff-8abe5aa8fdd8f398c83db7c61455bf18bfa9c2af7b0669a3203684d66f239300R37

File: config/services/accounts.yml (hunk 1 above)
Comment (sowbiba) [reply config/services/accounts.yml:9]: We need a task to upgrade it separately, will be probably included in the next major (?)

**Pull request merged**